packages/amplify-provider-awscloudformation/lib/system-config-manager.js

```diff
@@ -124,7 +126,10 @@ function getProfileCredentials(profileName) {
       }
     });
   }
-  return profileCredentials;
+  return {
+    accessKeyId: profileCredentials.aws_access_key_id,
+    secretAccessKey: profileCredentials.aws_secret_access_key,
+  };
```

Comment:
Why don't we want to return the region out here?

Reply:
this part has been updated.
region is in the config file, and credentials can be in the config file or the credentials file, although it's recommended to store it in the credentials file.
the refactor allows both, and jus normalize the key.